build/root/Makefile

```diff
@@ -408,7 +408,6 @@ define CMD_HELP_INFO
 #   make kubectl kube-proxy
 endef
 #TODO: make EXCLUDE_TARGET auto-generated when there are other files in cmd/
-#TODO: should we exclude the target "libs" but include "cmd/libs/go2idl/*"?
```

Comment:
this commit looks drastically different than it did previously... it was a strict rename before, now there are hundreds of changed lines

Reply:
and lots of files outside the `staging/src/k8s.io/{kube-gen => code-generator}/cmd/client-gen` path are affected

Reply:
Previously it fixed only what was needed to build in origin. Now, it works properly inside k8s, that's also the reason the rename commit is so big.

Reply:
would prefer the rename commit separate from the fixup response in the future...

Reply:
Will remember for next time, sorry was trying to fix that mess.